component-samples/demo/README.MD

```diff
@@ -209,25 +230,88 @@ The FDO PRI HTTP Java Device Sample currently supports `fdo_sys` and `fidoallian
 
     *exec-cb* - This command returns the status / progress of the process running.
 
+    Sample SVI instruction :
+  
+    `{"filedesc" : "sample.txt","resource" : "https://www.google.com"},
```

Comment:
Not sure if using google.com requires any disclaimer . can we just say "website address"

Reply:
Updated